db/migrations/20230103102027_create_users.sql

```diff
@@ -0,0 +1,17 @@
+-- migrate:up
+CREATE TABLE users (
+    id INT NOT NULL AUTO_INCREMENT,
+    name VARCHAR(100) NOT NULL,
+    password VARCHAR(100) NOT NULL,
```

Comment:
```suggestion
    password VARCHAR(200) NOT NULL,
```
bcrypt로 hash되는 값을 고려하여 넉넉하게 늘려주세요